### ISSUE TITLE
`:line-length` does not report `:end-row` or `:end-col`

### ISSUE THREAD

**Issue by colin-p-hill** [clj-kondo/clj-kondo#1951]
**version**

v2023.01.12

**platform**

Windows 11, native, scoop.

**editor**

Cursive, clojure-extras.

**problem**

The `:line-length` lint reports `:col` and `:row`, but not end locations. In my editor, this causes the entire rest of the file to be highlighted with a warning.

**repro**

```clojure
(ns repro)
(def this-is-a-very-long-symbol-aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa nil)
;; This is a very long comment aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```
```shell
> clj-kondo --lint repro.clj --config '{:output {:format :edn}}' | jet --pretty --query ':findings'
[{:col 81,
  :filename "repro.clj",
  :level :warning,
  :message "Line is longer than 80 characters.",
  :row 2,
  :type :line-length}
 {:col 81,
  :filename "repro.clj",
  :level :warning,
  :message "Line is longer than 80 characters.",
  :row 3,
  :type :line-length}]
```

**config**

```edn
{:linters {:line-length {:max-line-length 80}}}
```

**expected behavior**

I expect `:end-row` and `:end-col` to be present so that editors know where to end highlighting.

**Comment by borkdude** [comment]
I missed this one, sorry

**Comment by colin-p-hill** [comment]
No worries! I opened a bunch in a short time, easy to miss one.

**Comment by borkdude** [comment]
@colin-p-hill Is there any way for you to test this out? E.g. you could download a binary from Appveyor and set clojure-extras to use the binary, temporarily, just to test this fix.

**Comment by colin-p-hill** [comment]
@borkdude Works perfectly 😄

Before:
![image](https://user-images.githubusercontent.com/66083075/218865626-dbadfc01-f3da-4051-914c-7081ba0c22fd.png)

After:
![image](https://user-images.githubusercontent.com/66083075/218865538-2a5978b2-bcdc-44d7-b619-538b861438f1.png)
